src/Backups/BackupsWorker.cpp

```diff
@@ -948,7 +949,7 @@ BackupsWorker::makeBackupCoordination(bool on_cluster, const BackupSettings & ba
 
     bool is_internal_backup = backup_settings.internal;
 
-    String root_zk_path = context->getConfigRef().getString("backups.zookeeper_path", "/clickhouse/backups");
+    String root_zk_path = config->getString("backups.zookeeper_path", "/clickhouse/backups");
```

Comment:
Here if I am not mistaken we actually made `backups.zookeeper_path` not changeable without restart, however I think it made it uniform with the other settings that we use in the ctor, so it is better in my opinion.